examples/aws/example/models.py

```diff
@@ -8,16 +8,11 @@
 
 class AbstractExamplePage(Page, AutoPublishingWagtailBakeryModel):
     body = StreamField(
-        [
-            ('paragraph', blocks.RichTextBlock())
-        ],
+        [("paragraph", blocks.RichTextBlock())],
         use_json_field=True,
     )
 
-    content_panels = [
-        FieldPanel('title'),
-        FieldPanel('body')
-    ]
+    content_panels = [FieldPanel("title"), FieldPanel("body")]
```

Comment:
It's unusual to see `content_panels` bunched up on a single line. I'm sure Black thought it was a good idea, but I suggest adding a final `,` to tell it that we want it split up on multiple lines.

Reply:
Good point, missed this as I only glanced over the noise the Black created 😄